### ISSUE TITLE
Feature: Chart for Boolean true/false

### ISSUE THREAD

**Issue by debegr92** [thomasnordquist/MQTT-Explorer#418]
Please allow 'true' and 'false' to be plotted in a chart.
True and false should be plotted as 1/0 so that I can see a boolean sensor or button.

**Comment by thomasnordquist** [comment]
That is how it works:

<img width="1136" alt="Bildschirmfoto 2020-08-07 um 16 33 24" src="https://user-images.githubusercontent.com/7721625/89656334-bda6b000-d8cb-11ea-9f8c-0b3e9d8736e6.png">
The chart on the bottom is configured with chart-interpolation "step after"

**Comment by thomasnordquist** [comment]
Oh sry, now I got what you mean.
If the message has no json object but only `true` / `false`.

**Comment by debegr92** [comment]
Exactly, now you understood correct! So, if the payload is a JSON and has a boolean value, it is working, but as plain true/false would be great! This is a valid MQTT payload as well.
